examples/with-docker/Dockerfile

```diff
@@ -1,20 +1,20 @@
 # Install dependencies only when needed
-FROM node:14-alpine AS deps
```

Comment:
It might be good to update the docs to point to this example file so we aren't duplicating this, what do you think @molebox?

Also we should probably keep the node version locked by default so it doesn't change and break dependencies unexpectedly

Reply:
sgtm

Reply:
The new deployment docs remove the Dockerfile code and point directly to the example: https://github.com/vercel/next.js/pull/32006

Would appreciate reviews there!

Reply:
> Also we should probably keep the node version locked by default so it doesn't change and break dependencies unexpectedly

looks like the node version has been removed unintentionally